src/telegram/_reply.py

```diff
@@ -438,6 +449,7 @@ def __init__(
         message_id: int,
         chat_id: Optional[Union[int, str]] = None,
         allow_sending_without_reply: ODVInput[bool] = DEFAULT_NONE,
+        checklist_task_id: Optional[int] = None,
```

Comment:
this is a breaking change @aelkheir @Bibo-Joshi 

should be added to the end of the parameter list

Reply:
Damn, we missed that. Maybe we should change it to make it non breaking. With the infinite retry bug we would then have two bugs for a bug fix release

Reply:
would changing the order of the argument again make it breaking again 🤔 

we could just accept it was breaking and announce as such in the channel

Reply:
I agree with @aelkheir on this ...